instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/RequestAccess.java

```diff
@@ -24,6 +26,36 @@ protected RequestAccess computeValue(Class<?> type) {
         }
       };
 
+  @Nullable
+  static String getLambdaArn(Object request) {
+    RequestAccess access = REQUEST_ACCESSORS.get(request.getClass());
+    if (access.getLambdaConfiguration == null) {
+      return null;
+    }
+    try {
+      Object config = access.getLambdaConfiguration.invoke(request);
+      if (config == null) {
+        return null;
+      }
+      Method method = config.getClass().getMethod("getFunctionArn");
```

Comment:
we try to avoid doing reflective lookups on each method invocation

Reply:
you could do the `class.getMethod("getFunctionArn")` call only once and cache the Method/MethodHandle for that